cosmo_tester/test_suites/test_blueprints/nodecellar_test.py

```diff
@@ -171,7 +171,7 @@ def _assert_monitoring_data_exists(self):
         client = InfluxDBClient(self.env.management_ip, 8086, 'root', 'root',
                                 'cloudify')
         self._assert_mongodb_collector_data(client)
-        self.assert_monitoring_data_exists()
+        self.assert_deployment_monitoring_data_exists(self.deployment_id)
```

Comment:
and here, the self.deployment_id property doesn't really exist. the test doesn't specify a deployment id when deploying the blueprint, meaning that self.test_id will be used.